### DISCUSSION TITLE
AdvancedEnchantments Addon

### DISCUSSION THREAD

**Discussion by DeOpping** [QuickShop-Community/QuickShop-Hikari#1155] [Ideas]
Hello!
We recognize there is an addon for EcoEnchants, and that's great! EcoEnchants is a great plugin.
We would be using that, but it doesn't support Bedrock players. Bedrock players can't apply custom enchants via an anvil, that's just how Bedrock Edition works, not really a solution to that.

However, AdvancedEnchantments has a way to add enchant books to your items without an anvil, via drag & drop. That is why we are using AdvancedEnchantments instead.

I believe an addon for AdvancedEnchantments that does the same as the addon for EcoEnchants would be cool!

**Comment by Ghost-chu** [comment]
Does AA have any issues/conflict with QuickShop-Hiakri?

We make EE addon because the default item previewing doesn't show EE's custom enchantments.

**Comment by DeOpping** [comment]
Ah, okay that's understandable, I haven't checked if that's the case. I will respond once I do!

**Comment by DeOpping** [comment]
Yup! Seems to work fine, the enchantment book's display name is put on the sign!
